app/controllers/teams_controller.rb

```diff
@@ -14,6 +14,10 @@ def index
   # GET /teams/1
   # GET /teams/1.json
   def show
+    if @team.name.starts_with?("portus_global_team_")
```

Comment:
I think that you can just raise an ActiveRecord::NotFound or something like that and you wouldn't need the `not_found` method ;)